proposed/2021/DotnetNugetTrustedSigners.md

```diff
@@ -0,0 +1,196 @@
+        --interactive                   Allows command to stop and wait for user input or action.
+        --configfile                    The NuGet configuration file. If specified, only the settings from this file will be used. If not specified, the hierarchy of configuration files from the current directory will be used.
+
+EXAMPLE:
+    $ dotnet nuget trusted-signers list
+    Registered trusted signers:
+
+    1.   nuget.org [repository]
+        Service Index: https://api.nuget.org/v3/index.json
+        Certificate fingerprint(s):
+            SHA256 - 0E5F38F57DC1BCC806D8494F4F90FBCEDD988B46760709CBEEC6F4219AA6157D
+
+    2.   microsoft [author]
+        Certificate fingerprint(s):
+            SHA256 - 3F9001EA83C560D712C24CF213C3D312CB3BFF51EE89435D3430BD06B5D0EECE
+            SHA256 - AA12DA22A49BCE7D5C1AE64CC1F3D892F150DA76140F210ABD2CBFFCA2C18A27
+
+    3.   myUntrustedAuthorSignature [author]
+        Certificate fingerprint(s):
+            [U] SHA256 - 518F9CF082C0872025EFB2587B6A6AB198208F63EA58DD54D2B9FF6735CA4434
+```
+
+#### `dotnet nuget trusted-signers sync`
+
+```
+Requests the latest list of certificates used in a currently trusted repository to update the the existing certificate list in the trusted signer.
+
+Note: This gesture will delete the current list of certificates and replace them with an up-to-date list from the repository.
+
+USAGE:
+    dotnet nuget trusted-signers sync [OPTIONS] <name>
+
+OPTIONS:
+    -h, --help                          Prints help information.
+    -v, --verbosity <LEVEL>             Set the MSBuild verbosity level. Allowed values are q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic].
+        --interactive                   Allows command to stop and wait for user input or action.
+        --configfile                    The NuGet configuration file. If specified, only the settings from this file will be used. If not specified, the hierarchy of configuration files from the current directory will be used.
+```
+
+#### `dotnet nuget trusted-signers remove`
+
+```
+Removes any trusted signers that match the given name.
+
+USAGE:
+    dotnet nuget trusted-signers remove [OPTIONS] <name>
+
+OPTIONS:
+    -h, --help                          Prints help information.
+    -v, --verbosity <LEVEL>             Set the MSBuild verbosity level. Allowed values are q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic].
+        --interactive                   Allows command to stop and wait for user input or action.
+        --configfile                    The NuGet configuration file. If specified, only the settings from this file will be used. If not specified, the hierarchy of configuration files from the current directory will be used.
+```
+
+#### `dotnet nuget trusted-signers trust-package`
+
+```
+Adds a trusted signer with the given name to the config, based on one or more packages.
+
+USAGE:
+    dotnet nuget trusted-signers trust-package [OPTIONS] <name> <package>...
```

Comment:
I'm a little concerned that `name` and `package` may not click for customers right away. Would it be possible to add an example to the help output? For example: `dotnet nuget trusted-signers trust-package Contoso .\contoso.library.nupkg`